### PR TITLE
Upgrade terraform-provider-rabbitmq to v1.7.0

### LINKED ISSUE #88: Upgrade terraform-provider-rabbitmq to v1.7.0
Release details: https://github.com/cyrilgdn/terraform-provider-rabbitmq/releases/tag/v1.7.0

### PR DESCRIPTION
**Pull request by thomas11**
Three new data sources which don't seem to have docs:
  - Exchange
  - User
  - VHost

Resolves #88

### CODE REVIEW

**Review by guineveresaenger — APPROVED** [review]
The upstream docs do not document any data sources either so 🚢

File: provider/resources.go [hunk 1]
```diff
@@ -110,8 +117,18 @@ func Provider() tfbridge.ProviderInfo {
...
+				Tok: makeResource(mainMod, "OperatorPolicy"),
+				Docs: &tfbridge.DocInfo{
+					Source: "operator-policy.html.markdown",
+				},
+			},
+		},
+		DataSources: map[string]*tfbridge.DataSourceInfo{
+			"rabbitmq_exchange": {Tok: makeDataSource(mainMod, "getExchange")},
+			"rabbitmq_user":     {Tok: makeDataSource(mainMod, "getUser")},
+			"rabbitmq_vhost":    {Tok: makeDataSource(mainMod, "getVHost")},
```
Comment (guineveresaenger) [inline provider/resources.go:130]:
```suggestion
			"rabbitmq_vhost":    {Tok: makeDataSource(mainMod, "getVhost")},
```

^^ that spelling is weird but it's consistent with the way the snake_case maps to CamelCase

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
actually I take it back - found a nit

File: provider/resources.go (hunk 1 above)
Comment (guineveresaenger) [reply provider/resources.go:130]: then again, the corresponding Resource says `VHost` not `Vhost` so maybe it doesn't matter 😭

Comment (thomas11) [reply provider/resources.go:130]: Yeah I was going for consistency with the existing resource

**Review by guineveresaenger — APPROVED** [review]
:shipit: once the tests pass. :P Thank you!

**Pull request merged**